lighthouse-core/runner.js

```diff
@@ -46,7 +46,8 @@ class Runner {
       Sentry.captureBreadcrumb({
         message: 'Run started',
         category: 'lifecycle',
-        data: sentryContext && sentryContext.extra,
+        // TODO is this really needed? the 'extra' data is already set at the end of sentry.js ... why put in breadcrumb?
```

Comment:
previously you got a few nice UI perks while traversing the history of events but not sure if that's how it works with the new sdk and/or if they've changed things on the server UI